composer.json

```diff
@@ -4,7 +4,7 @@
     "type": "drupal-module",
     "description": "Apigee Edge for Drupal.",
     "require": {
-        "php": "^7.4 || ^8.0",
+        "php": "^8.0",
```

Comment:
Should this really be done as part of a bugfix release? I would say this type of change should be more at least a minor version upgrade, or even a major version.

My project is still (unfortunately) using PHP 7.4, and now I cannot upgrade to the security release published yesterday without upgrading my whole project to PHP8 (which has other dependencies).
Should this be a **hard requirement**?